core/templates/pages/exploration-editor-page/services/exploration-data.service.spec.ts

```diff
@@ -115,20 +146,26 @@ describe('Exploration data service', function() {
         return [];
       }
     });
-    $httpBackend.expect('GET', '/createhandler/data/0?apply_draft=true')
-      .respond(sampleDataResults);
+
     $httpBackend.expectPUT('/createhandler/autosave_draft/0').respond({
```

Comment:
It seems that the coverage has decreased because you're not flushing this request anymore. I tried to fix it but I don't know to deal with Angular http calls and AngularJS http calls in the same test. Do you know how to fix it? @bansalnitish

Reply:
We can issue http calls in the same test both via Angular and AngularJS I think. I think I've done in few tests. Could you check back? I'll try to fix this too.

Reply:
Did you fix it? @bansalnitish